### ISSUE TITLE
Empty ObjectString missing validation inside query schema

### ISSUE THREAD

**Issue by fernandohjunqueira** [elysiajs/elysia#505]
### What version of Elysia.JS is running?

0.8.17

### What platform is your computer?

Linux 6.7.6-arch1-1 x86_64

### What steps can reproduce the bug?

I've created an object to be passed as a query parameter to store the current page the user is in and the maximum number of items to display per page.

To do so, I've used ObjectString. If pagination is not sent or is not an object, the validation fails. However, if it is an empty object (or any kind of object at all) the validation does not fail.

I needed those two fields to be required and to be of type number.

Use the code and the URL below to reproduce:

```typescript
import { Elysia, t } from "elysia";

const lists = new Elysia({ prefix: "/lists" })
  .get("/", ({ query }) => {
    console.log({ query })
    return "Hello"
  }, {
    query: t.Object({
      pagination: t.ObjectString({
        pageIndex: t.Number(),
        pageLimit: t.Number()
      })
    })
  })

export { lists }    
```

Type in your browser: `http://localhost:3000/lists?pagination={}`

A "Hello" message will appear, although it was not correctly validated.

### What is the expected behavior?

The expected behavior was for it to fail if the param is missing, if the object is empty, or one of the fields is missing, or one or both the fields to not have a number value.

It only fails if the param is missing or is not an object.

### What do you see instead?

The page loads with the missing required values from ObjectString as undefined, instead of failing at validation.

**Comment by SaltyAom** [comment]
Fixed in [711925a](https://github.com/elysiajs/elysia/pull/465/commits/711925a5b5561452b58979b26917cb13a6cf8d9c) under `1.0.0-beta.14`

**Comment by SaltyAom** [comment]
Should be fixed in 1.0.0

Feels free to reopen if issue still persists.
